src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/App.razor

```diff
@@ -1,7 +1,6 @@
 ﻿<CascadingAuthenticationState>
```

Comment:
Is leaving in this cascading authentication state intentional even when no authentication is enabled?

Reply:
Yes, it's intentional, because there's no drawback to having it here if it's not doing anything.

However we had a whole discussion before about whether it could be eliminated from the app source code entirely, even if you are using authentication. I personally would still like it if we did that, but it's not urgent - we can ship 3.0 without doing that, and it would be a back-compatible change in the future to make it there by default with duplicates being a no-op.